avm/ptn/security/security-center/main.bicep

```diff
@@ -23,89 +23,288 @@ param deviceSecurityGroupProperties object = {}
 @description('Optional. Security Solution data.')
 param ioTSecuritySolutionProperties object = {}
 
+@description('Optional. The sub-plan selected for a Standard pricing configuration, when more than one sub-plan is available. Each sub-plan enables a set of security features. When not specified, full plan is applied. For VirtualMachines plan, available sub plans are "P1" & "P2", where for resource level only "P1" sub plan is supported. Only usable if PricingTier = "Standard".')
+@allowed([
+  'P1'
+  'P2'
+])
+param virtualMachinesSubPlan string?
+
 @description('Optional. The pricing tier value for VMs. Azure Security Center is provided in two pricing tiers: free and standard, with the standard tier available with a trial period. The standard tier offers advanced security capabilities, while the free tier offers basic security features. - Free or Standard.')
 @allowed([
   'Free'
   'Standard'
 ])
 param virtualMachinesPricingTier string = 'Free'
 
+@description('Optional. If set to "False", it allows the descendants of this scope to override the pricing configuration set on this scope (allows setting inherited="False"). If set to "True", it prevents overrides and forces this pricing configuration on all the descendants of this scope. This field is only available for subscription-level pricing.')
```

Comment:
I think tt's not always 100% correct to copy property description from the REST API spec:

`This field is only available for subscription-level pricing.` is kind of redunant because module's deployment scope is subcription.

Reply:
Touche 😄